src/Option_Alpha/web/routes/report.py

```
@@ -0,0 +1,112 @@
+    format: Annotated[str, Query(pattern=r"^md$")] = "md",  # noqa: A002
+) -> StreamingResponse:
+    """Download a generated analysis report for a debate.
+
+    Currently supports ``?format=md`` (GitHub-Flavored Markdown).
+    The report always includes the full legal disclaimer.
+
+    Raises HTTP 404 if the debate_id does not match any stored thesis.
+    """
+    # Fetch the thesis — the repository method searches by ticker,
+    # so we need to look up by the database row ID directly.
+    conn = repo._db.connection  # noqa: SLF001
```

Ruff (0.15.1)

[warning] Unused `noqa` directive (non-enabled: `SLF001`)

Remove unused `noqa` directive

(RUF100)